helpers/custom_console_logger.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
+
+/*
+ * Custom Console log methods. Apply different colors for different log levels/severity.
+ *
+ */
+//
+// const CONSOLE_RESET = "\x1b[0m";
```

Comment:
for colour logging you can enable this commented lines and disable below code.

Reply:
Done